tableauserverclient/models/interval_item.py

```diff
@@ -75,6 +75,20 @@ def from_xml_element(cls, parsed_response, frequency):
 
 class HourlyInterval(IntervalItem):
     def __init__(self, start_time, end_time, interval_occurrence, interval_value):
+        self._validate_time(start_time)
+        self._validate_time(end_time)
+
+        if interval_occurrence != IntervalItem.Occurrence.Hours and \
+                interval_occurrence != IntervalItem.Occurrence.Minutes:
+            error = "Invalid interval type defined: {}.".format(interval_occurrence)
+            raise ValueError(error)
+        elif interval_occurrence == IntervalItem.Occurrence.Hours and int(interval_value) not in [1, 2, 4, 6, 8, 12]:
```

Comment:
Similarly I'm not sure how this worked before, parsed responses will be strings, found via test failures.